.gitlab/source_test/linux.yml

```diff
@@ -148,6 +148,16 @@ lint_flavor_heroku_linux-x64:
   variables:
     FLAVORS: '--flavors heroku'
 
+lint_python:
+  stage: source_test
+  image: gcr.io/datadoghq/agent-circleci-runner:v31988376-bfbb3afb
+  tags: ["arch:amd64"]
+  needs: []
+  before_script:
+    - source /root/.bashrc
```

Comment:
What do we need to source there?

Reply:
Indeed this is maybe not necessary. I put it "just in case" because we do this most of the time. Will test without